frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -55,9 +55,14 @@ interface NewRunParametersProps {
   specParameters: SpecParameters;
   handlePipelineRootChange?: (pipelineRoot: string) => void;
   handleParameterChange?: (parameters: RuntimeParameters) => void;
+  handleValidInput?: (validinput: boolean) => void;
```

Comment:
This name is a bit ambiguously in terms of its purpose: Do you want to communicate with parent component that the input is valid? How about name `setIsValidInput(isValid: boolean)`

Reply:
This one is better! Thank you James.